backstage/templates/scaffolder/terraform-mediawiki/content/mkdocs.yml

```diff
@@ -0,0 +1,9 @@
+---
+site_name: '${{ values.name }} Documentation'
+
+nav:
+  - Home: README.md
+  - Usage: usage.md
+
```

Comment:
```suggestion
  - MediaWiki: mediawiki.md
```

Reply:
that suggestion might not be syntactically correct..

Reply:
done